### PR TITLE
Skipping the CNTK install in some travis jobs to speed up tests

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**
### PR Overview

- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [ ] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
It seems flaky tests invaded Travis land. Will we be able to reclaim our build? Will we take back what is ours? 

@fchollet what do you think should be our approach towards flaky tests? There are too many of them now. Even with a working PR, I would say that there is 50-75% that the build would fail (for multiple tests, it's not only the 10 minutes timeout anymore). It's really slowing down implementation and confusing our contributors. 
* Should I open an issue for each flaky test that comes up? 
* Should we run multiple times those flaky test to force them to fail with high probability to debug more easily? 
* Should we skip them?

File: tests/keras/backend/backend_test.py [hunk 1]
```diff
@@ -1425,28 +1425,37 @@ def test_bias_add(self):
         with pytest.raises(ValueError):
             K.bias_add(x, b, data_format='channels_middle')
 
-    def test_batchnorm(self):
```
Comment (taehoonlee) [inline tests/keras/backend/backend_test.py]: The skipping looks reasonable, however, it is hard for me to see improvements in these refactoring...

**Comment by fchollet** [comment]
> Should I open an issue for each flaky test that comes up?

I think so. Fixing a flaky test is usually not very hard, we just need to make sure all our tests are deterministic.

**Comment by gabrieldemarmiesse** [comment]
I opened an issue for each of the flaky test I could find. I tagged them `flaky test`.
https://github.com/keras-team/keras/issues?q=is%3Aissue+flaky+is%3Aopen+label%3A%22flaky+test%22

@fchollet 
@taehoonlee 
@farizrahman4u 
@Dref360

**Pull request merged**